src/main/java/com/redhat/cloud/notifications/db/BehaviorGroupResources.java

```diff
@@ -47,7 +48,8 @@
         return session.createQuery(query, BehaviorGroup.class)
                 .setParameter("accountId", accountId)
                 .setParameter("bundleId", bundleId)
-                .getResultList();
+                .getResultList()
+                .onItem().transform(behaviorGroups -> behaviorGroups.stream().distinct().collect(Collectors.toList()));
```

Comment:
I was getting the `BehaviorGroup` repeated, as many times as it has a `BehaviorGroupAction`

Reply:
Oops, sorry about that one, it should have been tested 😃

I took the liberty of pushing a commit to slightly change the fix. Your version did the job, thanks for finding out that we needed to use `distinct` there! I moved this at a lower level, during the Hibernate query execution. Unfortunately Hibernate Reactive does not support query hints right now so I had to use a `@NamedQuery` but I'll talk with the Hibernate team about that.